### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by mattfaltyn**
Updated bitcoin-s README doc with the following:
- table of contents 
- description of bitcoin-s
- production readiness
- trustworthiness of bitcoin-s
- getting started for non-devs
- getting set up for devs
- link to contributing guidelines
- good first issues
- license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,55 @@
...
-Feature-rich toolkit for making Bitcoin and Lightning applications
-on the JVM.
+Feature-rich toolkit for making Bitcoin and Lightning applications on the JVM.
 
-For a complete guide on how to get started with Bitcoin-S, see our website at
-[Bitcoin-S.org](https://bitcoin-s.org)
+For a complete guide on how to get started with Bitcoin-S, see our website at [Bitcoin-S.org](https://bitcoin-s.org).
 
-### Getting started
+Released under the terms of the [MIT LICENSE](LICENSE).
```
Comment (benthecarman) [inline README.md]: You mentioned the license twice, here and at the bottom, idk if we need both

**Review by benthecarman — COMMENTED** [review]
Looks overall good! Thanks

File: README.md [hunk 2]
```diff
@@ -1,15 +1,55 @@
 ![Bitcoin-S logo](website/static/img/bitcoin-s-dark-logo.png)
 [![Build Status](https://github.com/bitcoin-s/bitcoin-s/workflows/Release/badge.svg)](https://github.com/bitcoin-s/bitcoin-s/actions) [![Coverage Status](https://coveralls.io/repos/github/bitcoin-s/bitcoin-s/badge.svg?branch=master)](https://coveralls.io/github/bitcoin-s/bitcoin-s?branch=master) [![Maven Central](https://img.shields.io/badge/Maven%20Central-0.6.0-brightgreen.svg)](https://mvnrepository.com/artifact/org.bitcoin-s) [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/bitcoin-s-core)
```
Comment (Christewart) [inline README.md]: Maybe these shields can be updated too? I wish they would automatically update, but the latest release is 1.7.0

File: README.md (hunk 1 above)
Comment (mattfaltyn) [reply README.md]: Will remove the one at the top of the doc!

File: README.md (hunk 2 above)
Comment (mattfaltyn) [reply README.md]: > Maybe these shields can be updated too? I wish they would automatically update, but the latest release is 1.7.0

Will do!

**Review by benthecarman — APPROVED** [review]

**Pull request merged**